ios/PurchasesHybridCommon/PurchasesHybridCommonTests/Mocks/MockPurchases.swift

```diff
@@ -4,9 +4,94 @@
 //
 
 import Foundation
-import Purchases
+@testable import RevenueCat
+import StoreKit
 
 class MockPurchases: Purchases {
+
+    init() {
```

Comment:
a lot of this is taken from https://github.com/RevenueCat/purchases-hybrid-common/pull/113, other parts are just updating the API calls